models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
why not modify src/api/database/sequelize.js directly? Also the JS syntax of this file is inconsistent with the rest of the source files.

Reply:
This is auto-generated by Sequelize Cli after run `npx sequelize-cli init`